btree/btree.c

```diff
@@ -1015,7 +1056,9 @@ struct m0_btree_oimpl {
 	struct node_op  i_nop;
```

Comment:
Please write some small description for each of the member variables in this structure since the names do not make it obvious to the reader.
Also see if you can add comments to some other structures members too. Just one line per member variable is sufficient.